hackage-security/src/Hackage/Security/Trusted/TCB.hs

```diff
@@ -167,7 +167,10 @@ newtype SignaturesVerified a = SignaturesVerified { signaturesVerified :: a }
 -- | Errors thrown during role validation
 data VerificationError =
      -- | Not enough signatures signed with the appropriate keys
-     VerificationErrorSignatures TargetPath
+     VerificationErrorSignatures TargetPath -- ^ what were we verifying?
+                                 Integer    -- ^ threshold
```

Comment:
Haddockumenting constructor fields seems new in haddock 8.6 so we have to turn of haddocks for GHC < 8.6.